### DISCUSSION TITLE
ecommerce framework yaml not working

### DISCUSSION THREAD

**Discussion by langert** [pimcore/pimcore#12001] [Q&A / Support] [answered]
Is there a working example for ecommerce framework?

Using 
bin/console config:dump-reference pimcore_ecommerce_framework
generates a yaml that causes different errors:

`noShipping:          { price_calculator: { factory_id: PriceCalculatorFactory, modificators: '~' } }`
=> error on array to string conversion

```
# Placeholder values in this tenant attributes definition (locale: "%%locale%%") will be replaced by the given placeholder value (eg. "de_AT")
placeholders:

  # Example:
  placeholders:        { '%%locale%%': de_AT }
```

=> error (placeholder key is doubled)

```
token_managers: 
 name: ~
```
=> error (should not be null) - same for pricing_manager ...

What is the correct way to get a working example in a blank pimcore installation?

Thx, Timo

**Comment by dvesh3** [comment, marked as answer]
Hi @langert, I have fixed the 2 wrong examples you described [here ](https://github.com/pimcore/pimcore/pull/12148) and concerning required configurations, this is just a reference to give you the config structure, so you have to setup the required modules and adjust the config according to that. 

I have also added the note to the PR so it is clear that the generated configuration should be not used as is.

Please note this line from docs:  _While configuring the framework you'll get instant feedback on configuration errors._
